src/dotnet-setversion/Program.cs

```diff
@@ -2,44 +2,107 @@
 using System.IO;
 using System.Linq;
 using System.Xml.Linq;
+using CommandLine;
 
 namespace dotnet_setversion
 {
-    public static class Program
+    internal static class Program
     {
-        private static int Main(string[] args)
+        internal static int Main(params string[] args)
```

Comment:
What's the benefit of using `params` here?

Reply:
Easier calling from the tests. `Program.Main(version, csprojFile)` rather than `Program.Main(new[] {version, csprojFile})`